package.json

```diff
@@ -123,5 +123,8 @@
   "collective": {
     "type": "opencollective",
     "url": "https://opencollective.com/polished"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.0.0"
```

Comment:
Is there a reason this is in dependencies vs devDependencies?

Reply:
This dep is generated in build step. Take a look at dist files